docs/appendices/release-notes/5.10.2.rst

```diff
@@ -47,3 +47,9 @@ Fixes
 - The ``status`` field within the JSON payload returned from the HTTP root
   endpoint ``(/)`` now shows 503 instead of 200 if the cluster state is
   considered unavailable to match how the ``ok`` field behaves.
+
+- Fixed an issue where a filter was not pushed down to the source when the
+  filtered column was aliased as part of a view or sub-select. This could lead
+  to an unnecessary full table scan. e.g.::
+
+      SELECT * FROM (SELECT a as aliased FROM doc.tbl) t WHERE t.aliased > 1;
```

Comment:
I tried out this, and it was working before the fix:
```
"Rename[aliased] AS t",
    "  └ Collect[doc.t1 | [field1 AS aliased] | (field1 AS aliased > 1)]"
```
So, does the issue happen only when unnest is also involved?

Reply:
tbh i did not try on this example, i just used a sub-select because i could type it in as a one-liner.

Reply:
I removed the example.